ynr/apps/sopn_parsing/helpers/parse_tables.py

```diff
@@ -16,6 +16,10 @@
     "surname",
     "candidates surname",
     "other name",
+    "candidate surname",
```

Comment:
Might it be useful to split the items in this list out in to two lists?

```
FIRST_NAME_FIELDS = []
LAST_NAME_FIELDS = []
NAME_FIELDS = FIRST_NAME_FIELDS + LAST_NAME_FIELDS
```
I don't know if this would help, but it might be a good way to understand what column contains what type of name, especially when you're converting them to First Middle Last format later on?